front/lib/amplitude/back/index.ts

```diff
@@ -85,7 +85,7 @@ export async function populateWorkspaceProperties(workspace: Workspace) {
   }
   const amplitude = getBackendClient();
 
-  const subscription = await subscriptionForWorkspace(workspace);
+  const subscription = await subscriptionForWorkspace(workspace.id);
```

Comment:
We never pass directly the `id` but the `sId` unless we don't really have a choice. Could we refactor to pass the `sId` here? 🙏🏻

Reply:
agree, was not sure. I will do that.

Reply:
Thanks! 🙏🏻